social_django/models.py

```diff
@@ -43,6 +44,10 @@ def __str__(self):
 
     class Meta:
         app_label = "social_django"
+        if django.VERSION[0] < 6:
+            constraints = [models.CheckConstraint(check=~models.Q(uid=""), name="user_social_auth_uid_required")]
+        else:
+            constraints = [models.CheckConstraint(condition=~models.Q(uid=""), name="user_social_auth_uid_required")]
```

Comment:
It would be better to favor the non-deprecated way instead

```suggestion
        if django.VERSION >= (5, 1):
            constraints = [models.CheckConstraint(condition=~models.Q(uid=""), name="user_social_auth_uid_required")]
        else:
            constraints = [models.CheckConstraint(check=~models.Q(uid=""), name="user_social_auth_uid_required")]
```